mne/viz/_3d.py

```diff
@@ -521,14 +521,14 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     if not isinstance(info, Info):
         raise TypeError('info must be an instance of Info, got %s'
                         % type(info))
-    if not (source is None) ^ (bem is None):
-        raise ValueError('source must be None if bem is not None '
-                         '(and vice versa)!')
+    if not (source is None) ^ (bem is None):  # XOR
+        raise ValueError('Only one of "source" and "bem" must be provided, '
+                         'got %s and %s respectively.' % (source, bem))
```

Comment:
But having `source is None and bem is None` is okay, right? What you have here forces exactly one to be specified.